common/src/main/java/org/wso2/testgrid/common/Utils.java

```diff
@@ -80,6 +81,59 @@ public static boolean executeCommand(String command, File workingDirectory) thro
         }
     }
 
+    /**
+     * Executes a command.
+     * Used to execute a script with given deployment details as environment variables.
+     *
+     * @param command          Command to execute.
+     * @param workingDirectory Directory the command is executed.
+     * @param deployment       Deployment details for environment.
+     * @return Returns the output of script execution as a String.
```

Comment:
`Returns` is not required since it is already specified in the JavaDoc

Reply:
Done